crates/tauri-plugin-android-update/permissions/autogenerated/reference.md

```
@@ -0,0 +1,74 @@
+## Default Permission
+
+This permission set enables the Android update workflow.
+
+#### Granted Permissions
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)